data/yaml/mspass.yaml

```diff
@@ -330,13 +334,13 @@ Metadata:
         readonly: false
       site_id:
         collection: wf_TimeSeries
-        readonly: false
+        readonly: true
```

Comment:
We (with Jinxin) decided to set these `"collection"_id` keys as writable for a reason. These keys correspond to the same attributes in the wf collection, which links a wf record to certain site/channel/source records. They are different from the actual `_id` keys, and we need to be able to change them to correctly associate a newly created wf record. If I remember right, there are codes that rely on this field to do save correctly.